### ISSUE TITLE
Proxmox v7 : API template field changed

### ISSUE THREAD

**Issue by stanlog** [xezpeleta/Ansible-Proxmox-inventory#40]
Hello, just for information, there is a change between Proxmox VE v6 an v7 API. Now in v7 "template" field in qemu exists if it's a template. In v6, there was an empty template field if it was not a template. For this reason, 
```
def get_names(self):
        if self.ver >= 4.0:
            return [vm['name'] for vm in self if vm['template'] != 1]
```
doesn't work anymore in v7.
Regards

**Comment by maynero** [comment]
@stanlog I already opened a PR #39 to fix this, once that's merge it should be able to handle that.

**Comment by stanlog** [comment]
Hello,

yes, sorry, I readed your PR after publishing my issue :( I'm going to 
close it.

Have a nice day.

Le 03/08/2021 à 07:00, Rowel a écrit :

**Comment by stanlog** [comment]
Close issue, 'cause a PR is already submitted.

**Comment by xezpeleta** [comment]
Fixed in #39
